tools/plugins/com.liferay.ide.layouttpl.ui/src/com/liferay/ide/layouttpl/ui/editor/LayoutTplEditor.java

```diff
@@ -289,6 +294,37 @@ private boolean isBootstrapStyle()
         return retval;
     }
 
+    private boolean is62()
+    {
+        try
+        {
+            final IProject project = getFile().getProject();
+
+            if( ProjectUtil.isMavenProject( project ) )
+            {
+                return true;
```

Comment:
THere are maven projects that aren't 6.2.  Instead lets check the deployment descriptors (liferay-layout-template.xml) instead.